response_operations_ui/templates/conversation-view/cv-back-link.html

```diff
@@ -1 +1,4 @@
+{% if thread_data.category != 'SURVEY' %}
```

Comment:
This feels wrong to me, and down to the fact we are passing something we shouldn't, line 186 in messages.py  shouldnt pass it, if not needed

Reply:
This has been updated and removed.